### PR TITLE
[VFX] Staging to Master

### PR DESCRIPTION
**Pull request by PaulDemeulenaere**
### Purpose of this PR

Staging PR to sync with internal VFX changes.

**Fixed**
- Move MinMaxAttribute to internal [#292](https://github.cds.internal.unity3d.com/unity/vfx-graphics/pull/292) @ludovic-theobald 
- Bounds helper feature polish [#263](https://github.cds.internal.unity3d.com/unity/vfx-graphics/pull/263) @ludovic-theobald 
- Collision with Depth + Physical camera is broken [#270](https://github.cds.internal.unity3d.com/unity/vfx-graphics/pull/270) @ludovic-theobald 

---
### Testing status
Yamato ⏳ 
[Yamato Internal](https://yamato.cds.internal.unity3d.com/jobs/339-ScriptableRenderPipelinePrivate/tree/master/.yamato%252Fall-vfx.yml%2523VFX_trunk/9657897/job) ⏳ 

---
### Comments to reviewers
Updated all VFX & Applied the formatting

### CODE REVIEW

File: com.unity.visualeffectgraph/Editor/Models/Contexts/Implementations/VFXBasicInitialize.cs [hunk 1]
```diff
@@ -84,26 +85,30 @@ protected override void GenerateErrors(VFXInvalidateErrorReporter manager)
...
                     }
                 }
-                catch { /* do nothing*/ }
+
+                if (data.boundsMode == BoundsSettingMode.Automatic)
+                {
+                    manager.RegisterError("CullingFlagAlwaysSimulate", VFXErrorType.Warning,
+                        "Setting the system Bounds Mode to Automatic will switch the culling flags of the Visual Effect asset" +
+                        " to 'Always recompute bounds and simulate'.");
+                }
```
Comment (PaulDemeulenaere) [inline com.unity.visualeffectgraph/Editor/Models/Contexts/Implementations/VFXBasicInitialize.cs:111]: @ludovic-theobald @julienamsellem I had to resolve a conflict here, can you double check this change ⬆️

**Comment by PaulDemeulenaere** [comment]
N.B.: We can expect failure in Smoke Test due to infinite import of material, see on [master](https://unity-ci.cds.internal.unity3d.com/job/9711293)

**Review by julienf-unity — APPROVED** [review]
Another vfx assets update will probably be required when https://github.com/Unity-Technologies/Graphics/pull/5125 lands.

**Comment by PaulDemeulenaere** [comment]
> Another vfx assets update will probably be required when #5125 lands.

Done at 8065aeafeb57ed1bf82eaa8e4a2091763e15f23c

**Review by ludovic-theobald — APPROVED** [review]

**Review by julienamsellem — APPROVED** [review]
Tested multi-vfx windows locally and it's all good.

**Pull request merged**